pkgs/development/libraries/mesa/default.nix

```diff
@@ -59,7 +58,7 @@ let
   driDrivers =
     (if dri_ == null
       then optionals (elem "drm" eglPlatforms) defaultDriDrivers
-      else dri_) ++ lib.optional stdenv.isLinux "swrast";
```

Comment:
`ERROR:  Problem encountered: The nine state tracker requires gallium softpipe/llvmpipe` so prefer gallium `swrast` as having `swrast` in both `galliumDrivers` and `driDrivers` is bad [1].

[1] https://gitlab.freedesktop.org/mesa/mesa/blob/master/meson.build#L208